modules/oe_content_sub_entity/modules/oe_content_sub_entity_person/tests/src/Kernel/PersonReferenceFormatterTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_sub_entity_person\Kernel;
+
+use Drupal\KernelTests\Core\Entity\EntityKernelTestBase;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+use Drupal\Tests\node\Traits\NodeCreationTrait;
+use Drupal\Tests\oe_content\Traits\EntityReferenceRevisionTrait;
+use Drupal\Tests\sparql_entity_storage\Traits\SparqlConnectionTrait;
+
+/**
+ * Tests Persons reference formatter.
+ */
+class PersonReferenceFormatterTest extends EntityKernelTestBase {
+
+  use NodeCreationTrait;
+  use EntityReferenceRevisionTrait;
+  use ContentTypeCreationTrait;
+  use SparqlConnectionTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
```

Comment:
`protected static $modules` check parent. Also, check if you need all the modules. Like `oe_content_sub_entity_document_reference ` for example

Reply:
Yes, we need all these modules for Person CT.